src/config/index.ts

```diff
@@ -466,6 +466,8 @@ export const OVERSEAS_ENTITY_SECTION_HEADING = "Overseas entity details";
 // REGISTRATION URLS WITH PARAMS
 export const ACTIVE_SUBMISSION_BASE_PATH = `transaction/:${ROUTE_PARAM_TRANSACTION_ID}/submission/:${ROUTE_PARAM_SUBMISSION_ID}/`;
 export const REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + ACTIVE_SUBMISSION_BASE_PATH;
+export const SOLD_LAND_FILTER_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL + SOLD_LAND_FILTER_PAGE;
+export const SECURE_REGISTER_FILTER_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL + SECURE_REGISTER_FILTER_PAGE;
```

Comment:
Is this definition required at this point as story is only concerned with the 'sold land filter' screen?